app/models/supplejack_api/concerns/record.rb

```diff
@@ -47,7 +47,10 @@ def self.find_multiple(ids)
       records = []
       records += active.find(string_ids) if string_ids.present?
       records += active.where(:record_id.in => integer_ids)
-      records = records.sort_by { |r| ids.find_index(r.record_id) || 100 }
+
+      records.sort_by do |record|
```

Comment:
Simpler solution written in [commit](https://github.com/DigitalNZ/supplejack_api/pull/288/commits/b54c68327d2a5617ae966d45cdcc656c7c8527c6) removed in favour of this as we have to take in account that ids can be either record_ids or ids or a combination. This way the sort_by will always find the index considering integer and string.